src/pageElements/homeLayout/cookiePopupWindowPage/cookiePopupWindowPage.tsx

```diff
@@ -36,9 +37,9 @@ const CookiePopupWindowPage = () => {
 				</div>
 				<div className={styles.textContentBlock__subtitle}>
 					Все на сайте — для вас,{" "}
-					<a href={cookieLinkTitle} target="_blank" rel="noreferrer" className={styles.textContentBlock__link}>
+					<Link href={"/agreement"} className={styles.textContentBlock__link} onClick={handleClick}>
```

Comment:
href={MainPath.Agreement}

create constant Agreement with this path in MainPath with src/services/router/routes.ts